demos/cms/cms_ver.c

```diff
@@ -69,6 +116,9 @@ int main(int argc, char **argv)
     fprintf(stderr, "Verification Successful\n");
```

Comment:
Existing bug where it writes to stderr not stdout?

Reply:
fixed.